<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="2901" started-at="2020-01-27T13:43:01 IST" finished-at="2020-01-27T13:43:04 IST">
    <groups>
    </groups>
    <test name="CreateCustomerAPITest" duration-ms="1803" started-at="2020-01-27T13:43:01 IST" finished-at="2020-01-27T13:43:03 IST">
      <class name="com.w2a.APITestingFrameWork.TestCases.CreateCustomerTest">
        <test-method status="PASS" signature="validateCreateCustomerAPIWithValidSecretKey(java.util.Hashtable)[pri:0, instance:com.w2a.APITestingFrameWork.TestCases.CreateCustomerTest@58d75e99]" name="validateCreateCustomerAPIWithValidSecretKey" duration-ms="1762" started-at="2020-01-27T13:43:01 IST" data-provider="data" finished-at="2020-01-27T13:43:03 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{email=trainer@manish.com", name=manish, description=Adding the first customer in framework}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateCreateCustomerAPIWithValidSecretKey -->
        <test-method status="PASS" signature="validateCreateCustomerAPIWithValidSecretKey(java.util.Hashtable)[pri:0, instance:com.w2a.APITestingFrameWork.TestCases.CreateCustomerTest@58d75e99]" name="validateCreateCustomerAPIWithValidSecretKey" duration-ms="1765" started-at="2020-01-27T13:43:01 IST" data-provider="data" finished-at="2020-01-27T13:43:03 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{email=trainer@manish1.com, name=manish K, description=Adding manishk1 the first customer in framework}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateCreateCustomerAPIWithValidSecretKey -->
      </class> <!-- com.w2a.APITestingFrameWork.TestCases.CreateCustomerTest -->
    </test> <!-- CreateCustomerAPITest -->
    <test name="DeleteCustomerAPITest" duration-ms="807" started-at="2020-01-27T13:43:03 IST" finished-at="2020-01-27T13:43:04 IST">
      <class name="com.w2a.APITestingFrameWork.TestCases.DeleteCustomerTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.w2a.APITestingFrameWork.TestCases.DeleteCustomerTest@1de5f259]" name="setUp" is-config="true" duration-ms="316" started-at="2020-01-27T13:43:01 IST" finished-at="2020-01-27T13:43:01 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="deleteCustomer(java.util.Hashtable)[pri:0, instance:com.w2a.APITestingFrameWork.TestCases.DeleteCustomerTest@1de5f259]" name="deleteCustomer" duration-ms="795" started-at="2020-01-27T13:43:03 IST" data-provider="data" finished-at="2020-01-27T13:43:04 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{id=cus_GceRBeF2vAlgI8}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[ID key is not present in JSON Response expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: ID key is not present in JSON Response expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at com.w2a.APITestingFrameWork.TestCases.DeleteCustomerTest.deleteCustomer(DeleteCustomerTest.java:39)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteCustomer -->
        <test-method status="FAIL" signature="deleteCustomer(java.util.Hashtable)[pri:0, instance:com.w2a.APITestingFrameWork.TestCases.DeleteCustomerTest@1de5f259]" name="deleteCustomer" duration-ms="801" started-at="2020-01-27T13:43:03 IST" data-provider="data" finished-at="2020-01-27T13:43:04 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[{id=cus_GceRCdGCB8ri6T}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[ID key is not present in JSON Response expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: ID key is not present in JSON Response expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at com.w2a.APITestingFrameWork.TestCases.DeleteCustomerTest.deleteCustomer(DeleteCustomerTest.java:39)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteCustomer -->
        <test-method status="PASS" signature="testDown()[pri:0, instance:com.w2a.APITestingFrameWork.TestCases.DeleteCustomerTest@1de5f259]" name="testDown" is-config="true" duration-ms="0" started-at="2020-01-27T13:43:04 IST" finished-at="2020-01-27T13:43:04 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDown -->
      </class> <!-- com.w2a.APITestingFrameWork.TestCases.DeleteCustomerTest -->
    </test> <!-- DeleteCustomerAPITest -->
  </suite> <!-- Suite -->
</testng-results>
